### PR TITLE
feat: Initial generation of google-cloud-binary_authorization-v1beta1

### PR DESCRIPTION
**Pull request by dazuma**

### CODE REVIEW

File: google-cloud-binary_authorization-v1beta1/lib/google/cloud/binary_authorization/v1beta1/binauthz_management_service_v1_beta1.rb [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        #
+        # * {::Google::Cloud::BinaryAuthorization::V1beta1::Policy Policy}
+        # * {::Google::Cloud::BinaryAuthorization::V1beta1::Attestor Attestor}
+        #
+        # To load this service and instantiate a client:
+        #
+        #     require "google/cloud/binary_authorization/v1beta1/binauthz_management_service_v1_beta1"
+        #     client = ::Google::Cloud::BinaryAuthorization::V1beta1::BinauthzManagementServiceV1Beta1::Client.new
+        #
+        module BinauthzManagementServiceV1Beta1
```
Comment (quartzmo) [inline google-cloud-binary_authorization-v1beta1/lib/google/cloud/binary_authorization/v1beta1/binauthz_management_service_v1_beta1.rb:47]: Do you want to remove `V1Beta1` from this `BinauthzManagementServiceV1Beta1` module name?

Comment (dazuma) [reply google-cloud-binary_authorization-v1beta1/lib/google/cloud/binary_authorization/v1beta1/binauthz_management_service_v1_beta1.rb:47]: Oh my, thanks for catching this. I need to pay attention and look for these when I generate these libraries.

Comment (dazuma) [reply google-cloud-binary_authorization-v1beta1/lib/google/cloud/binary_authorization/v1beta1/binauthz_management_service_v1_beta1.rb]: Fixed.

File: google-cloud-binary_authorization-v1beta1/synth.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    generator_args={
+        "ruby-cloud-gem-name": "google-cloud-binary_authorization-v1beta1",
+        "ruby-cloud-title": "Binary Authorization V1beta1",
+        "ruby-cloud-description": "Binary Authorization is a service on Google Cloud that provides centralized software supply-chain security for applications that run on Google Kubernetes Engine (GKE) and GKE on-prem.",
+        "ruby-cloud-env-prefix": "BINARY_AUTHORIZATION",
+        "ruby-cloud-grpc-service-config": "google/cloud/binaryauthorization/v1beta1/binaryauthorization_grpc_service_config.json",
+        "ruby-cloud-product-url": "https://cloud.google.com/binary-authorization/",
+        "ruby-cloud-api-id": "binaryauthorization.googleapis.com",
+        "ruby-cloud-api-shortname": "binaryauthorization",
+        "ruby-cloud-service-override": "BinauthzManagementServiceV1Beta1=BinauthzManagementService",
```
Comment (quartzmo) [inline google-cloud-binary_authorization-v1beta1/synth.py:38]: 👍

**Review by quartzmo — APPROVED** [review]

**Pull request merged**